Microsoft.Toolkit.Uwp.SampleApp/SamplePages/Menu/MenuPage.xaml

```diff
@@ -26,21 +28,22 @@
                        TooltipPlacement="{Binding TooltipPlacement.Value, Mode=TwoWay}">
 
             <controls:MenuItem x:Name="FileMenu"
-                               controls:Menu.InputGestureText="Alt+F">
+                               controls:Menu.InputGestureText="Alt+F"
+                               Header="^File">
```

Comment:
Noticed the SymbolIcon Header example was removed, can we not support both with the new ^?